hazelcast/src/test/java/com/hazelcast/cache/impl/CachePartitionEventDataTest.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
There is already a test called CacheSerializationTest, why not add this to that test?